Makefile

```diff
@@ -52,6 +52,7 @@ CHROMIUM_CLEAR_CACHE=false
 CHROMIUM_CLEAR_COOKIES=false
 CHROMIUM_DISABLE_JAVASCRIPT=false
 CHROMIUM_DISABLE_ROUTES=false
+GOTENBERG_BUILD_DEBUG_DATA=true
```

Comment:
Just need to put it below the `GOTENBERG_GRACEFUL_SHUTDOWN_DURATION` arg 😄

Reply:
np!